pkg/print/config.go

```diff
@@ -26,3 +27,26 @@ func ConfigValidationErrors(log *logger.CLILogger, groupValidationErrors []confi
 	log.FinishSpinnerWithError()
 	log.Errorf(sb.String())
 }
+
+func PreflightErrors(log *logger.CLILogger, results []*preflight.UploadPreflightResult) {
+	var s strings.Builder
+	s.WriteString("\nPreflight check results (state - title - message)")
+	for _, result := range results {
+		s.WriteString(fmt.Sprintf("\n%s - %q - %q", preflightState(result), result.Title, result.Message))
+	}
+	log.Info(s.String())
+}
+
+func preflightState(r *preflight.UploadPreflightResult) string {
```

Comment:
ditto

Reply:
I'll DRY this out in favour of `preflights.PreflightState`